### DISCUSSION TITLE
Templates for Project-Book-Exports to get started

### DISCUSSION THREAD

**Discussion by MaPeras** [Zettlr/Zettlr#4370] [Q&A] [answered]
Hi everyone,

I am experimenting with the project feature. The goal is to create a book out of several Markdown-Documents exported from pubpub.org - this would be a great tool for scholar-led-publishing.

Are there any LaTeX-Templates and YAML-Snippets floating around, that work great for such a setting? I am willing to include those in the Zettlr Documentation if that helps. But for now, the Documentation is rather brief regarding this.

Thanks a lot!
Max

**Comment by UFOMelkor** [comment, marked as answer]
I guess [eisvogel](https://github.com/Wandmalfarbe/pandoc-latex-template) is one of the most used templates in this field. It can be used for books too.

**Reply by MaPeras** [reply]
thank you, that's a great start! I had heard of it, but didn't know the documentation is so great. However, it doesn't seem to be [ideal for edited volumes](https://github.com/Wandmalfarbe/pandoc-latex-template/issues/341).
